aten/src/ATen/native/quantized/cpu/qdropout.cpp

```diff
@@ -0,0 +1,21 @@
+#include <ATen/ATen.h>
+#include <ATen/NativeFunctions.h>
+#include <torch/library.h>
+#include <ATen/quantized/Quantizer.h>
+#include <ATen/native/quantized/cpu/quantized_ops.h>
+
+namespace at {
+namespace native {
+
+DEFINE_DISPATCH(qdropout_stub);
+
+Tensor quantized_dropout(
```

Comment:
I think this op should take a `training` argument, like https://github.com/pytorch/pytorch/blob/master/torch/nn/functional.py#L1279

Reply:
nit: output_scale and output_zero_point should be the last two arguments of this op, otherwise looks good

Reply:
for the other parameters it would be best to match the order of original dropout op as well

Reply:
actually what is scale and input_scale?